charts/castai-db-optimizer/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 name: castai-db-optimizer
 description: CAST AI database cache deployment.
 type: application
-version: 0.37.0
+version: 0.37.1-rc.1
```

Comment:
I thought we gave up on rc builds?

Reply:
I'm gunshy after yesterday. I don't want this to risk getting into the wild